packages/common/src/components/Button/elements.ts

```diff
@@ -70,7 +70,8 @@ const getColor = ({
   return 'white';
 };
 
-const getHoverColor = ({ secondary }: OptionProps) => {
+const getHoverColor = ({ secondary, disabled }: OptionProps) => {
+  if (disabled) return '';
```

Comment:
@CompuIves I think it's better to use curly braces around the returns

```suggestion
  if (disabled) {
    return '';
  }
```

Reply:
I prefer to keep consistency here, I either change this here and all the other lines as well, or I do that in a separate PR. I prefer to hold it off this PR.